src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -52,29 +71,51 @@ const transformedLicenseRequest = {
   notifications: [],
 };
 
+const assignmentsData = {
```

Comment:
nit: if this object is about a singular assignment, should the variable name be `assignmentData`?

```suggestion
const assignmentData = {
```